charts/super-agent/charts/super-agent-deployment/tests/__snapshot__/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
```suggestion
**
```

Reply:
yes the second has this as base, I would merge that into this, I am not sure how it would work with ci/cd, if it fails we can break it down again

Reply:
we can merge https://github.com/newrelic/helm-charts/pull/1410 here

Reply:
I ended up doing it directly